v2/rules/images/banned-ports.yaml

```
@@ -0,0 +1,26 @@
+config-type: rule
+id: "images-banned-ports"
+name: "Banned Ports"
+path: "banned-ports.rego"
+
+description: |
+  Ensures that the container image does not expose ports that are disallowed by organizational policy.
+  The rule examines properties in the SBOM metadata and checks each value (expected in the format "port/protocol") against a provided banned ports list.
+  It fails if any banned port is exposed or if no banned ports list is provided.
+
+labels:
+  - "SBOM"
+  - "Images"
+
+level: "error"
+
+evidence:
+  content_body_type: "cyclonedx-json"
+  target_type: "container"
+  filter-by:
+  - target
+  - product
+with:
+  banned_ports:
+  - port: "3000"
+    protocol: "tcp"
```

I'd comment out such defaults, if we can
```suggestion
#   banned_ports:
#   - port: "3000"
#     protocol: "tcp"
```

---

Or maybe better something like this:
```suggestion
  banned_ports: [] # [{port: 3000, protocol: tcp}]
```